build/docker/intel-qat-initcontainer.Dockerfile

```
@@ -0,0 +1,35 @@
+# Copyright 2022 Intel Corporation. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM busybox as builder
+
+ARG DIR=/qat-init
+WORKDIR $DIR
+
+
+RUN (echo '#!/bin/sh' \
+&& echo 'for dev in `for pf in $ENABLING_PF_PCI_IDs; do lspci | awk -v dev="$pf" '"'"'$0 ~ dev {print "0000:" $1}'"'"'; done`; do'\
+&& echo '  DEVPATH="/sys/bus/pci/devices/$dev"'\
+&& echo '  NUMVFS="$DEVPATH/sriov_numvfs"'\
+&& echo '  echo $pf'\
+&& echo '  if [ -w "$NUMVFS" -a $(cat "$NUMVFS") -eq 0 ]; then'\
+&& echo '    cat "$DEVPATH/sriov_totalvfs" | tee "$NUMVFS"'\
+&& echo '    ls -la $NUMVFS'\
+&& echo '  fi'\
+&& echo 'done') > test.sh && cat test.sh && chmod +x test.sh
+
+FROM busybox
+COPY --from=builder /qat-init/test.sh /usr/bin/test.sh
+ENTRYPOINT ["/usr/bin/test.sh"]
```

I don't see the need for `builder`
```suggestion
FROM busybox
ADD /<script from repo path> /usr/bin/test.sh
ENTRYPOINT ["/usr/bin/test.sh"]
```
see idxd-config initcontainer.
```

---

Ah, I didn't know that I kept it like this.. Let me change! Thanks!